src/s3ql/umount.py

```diff
@@ -89,14 +89,27 @@ def get_cmdline(pid):
     cannot be determined for other reasons, log warning
     and return None.
     '''
+    
+    output = None
+    if os.path.isdir('/proc'):
+        cmd_path = '/proc/%d/cmdline' % pid
+        if os.path.isfile(cmd_path):
+            try:
+                with open(cmd_path, 'r') as cmd_file:
+                    output = cmd_file.read()
```

Comment:
Why not simply `return cmd_file.read()`?

Reply:
Well, I saw cases on my own system where `/proc/<pid>/cmdline` was empty (I guess that's for early bootstrapping processes?). I thought it would make sense to handle this case and return `None` instead of an empty string. 

But now, when looking at the code again how `get_cmdline` is being used, I guess it actually makes more sense to return the empty string in that case and only return `None` in case the cmdline could not be determined if either the process does not exist anymore or in case of an error.

TL;DR: Changed accordingly.